docs/reference/commandline/run.md

```diff
@@ -186,6 +186,64 @@ flag makes Docker attempt to create a new file and write the container ID to it.
 If the file exists already, Docker returns an error. Docker closes this
 file when `docker run` exits.
 
+### <a name="pid"></a> PID settings (--pid)
+
+```text
+--pid=""  : Set the PID (Process) Namespace mode for the container,
+             'container:<name|id>': joins another container's PID namespace
+             'host': use the host's PID namespace inside the container
+```
+
+By default, all containers have the PID namespace enabled.
+
+PID namespace provides separation of processes. The PID Namespace removes the
+view of the system processes, and allows process ids to be reused including
+PID 1.
+
+In certain cases you want your container to share the host's process namespace,
+basically allowing processes within the container to see all of the processes
+on the system. For example, you could build a container with debugging tools
+like `strace` or `gdb`, but want to use these tools when debugging processes
+within the container.
+
+#### Example: run htop inside a container
+
+Create this Dockerfile:
+
+```dockerfile
+FROM alpine:latest
+RUN apk add --update htop && rm -rf /var/cache/apk/*
```

Comment:
We should use `apk add --no-cache` here, so that we don't have to deal with the `rm -rf`.


Honestly, we could even omit the `docker build` at all (alpine is really fast with these steps), if we want to keep it simple.

Reply:
Good idea. One thing that came to my mind, I noticed that this option shares the VM process namespace when using DD for instance. Might be worth calling out?